main.py

```diff
@@ -1,7 +1,47 @@
 import init_django_orm  # noqa: F401
-
 from django.db.models import QuerySet
+from db.models import Genre, Actor
 
 
 def main() -> QuerySet:
```

Comment:
According to the checklist, you shouldn't have this fragment in your code: `if __name__ == "__main__": main()`. However, your `main` function is present without this fragment. It's not clear how it's supposed to be called. Please ensure that your code is organized correctly.